### DISCUSSION TITLE
Swiper slider with coverflow effect - problem with links

### DISCUSSION THREAD

**Discussion by Shc0d** [nolimits4web/swiper#7733] [Q&A]
Hello! There is a slider with the coverflow effect. Each slide has a link, however, the slides on the left and right are not clickable until you scroll them (3-4 slides). This happens exactly in Firefox and Safari.

Example: https://www.nestseekers.com

The problem is old, but still not solved (now I use Swiper 11.1.12). A similar one is described here: https://searchengines.guru/go?link=https://github.com/kidjp85/react-id-swiper/issues/145

Tell me how this can be fixed. Thank you!

**Comment by Shc0d** [comment]
Help please!

**Reply by dalvinxo** [reply]
- ### Yfvb t vc v o ce" r 9v~~ x. Xvn.    C@ v nzvn x.cxx # - [ ] # c@ex. Xf~~ h on.  Fde~~ ~~vccf v.  Creo xd creo cn f
